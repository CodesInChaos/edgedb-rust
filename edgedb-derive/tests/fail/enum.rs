use edgedb_derive::Queryable;

#[derive(Queryable)]
enum MyEnum {
    SomeValue,
    Option2,
}

fn main() {
}
