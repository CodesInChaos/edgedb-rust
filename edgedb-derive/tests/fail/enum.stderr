error: expected `struct`
 --> $DIR/enum.rs:4:1
  |
4 | enum MyEnum {
  | ^^^^
